app/controllers/practices_controller.rb

```diff
@@ -1,25 +1,24 @@
 class PracticesController < ApplicationController
   include CropperUtils
-  before_action :set_practice, only: [:show, :edit, :update, :destroy, :planning_checklist,
+  before_action :set_practice, only: [:show, :edit, :update, :destroy,
                                       :commit, :committed, :highlight, :un_highlight, :feature,
```

Comment:
we can remove `:commit` and `:committed` here too! :)

Reply:
Done